--- a/include/g_local.h
+++ b/include/g_local.h
@@ -77,7 +77,7 @@
 #define MOD_NAME				("KTX")
 #define MOD_FULLNAME			("KTX: Kombat Teams eXtreme")
 #define MOD_VERSION				("1.47")
-#define MOD_BUILD_DATE			(__DATE__ ", " __TIME__)
+#define MOD_BUILD_DATE			("May 16 2026, 20:38:52")
 #define MOD_SERVERINFO_MOD_KEY	("ktxver")
 #define MOD_URL					("https://github.com/QW-Group/ktx")
 #define GIT_COMMIT				("")
